packages/main/src/utils/discovery.config.ts

```diff
@@ -0,0 +1 @@
+export const queries = [{ featureType: 'capability', match: 'mrtd.*' }]
```

Comment:
This seems to be hard-coded (i.e. all deployed Service Agents will do this query when connecting to other agents). How can I as a Service Agent user configure my agent to use this or other queries (or none)?